cas-server-webapp/src/main/webapp/WEB-INF/view/jsp/protocol/openid/casOpenIdServiceFailureView.jsp

```diff
@@ -17,5 +17,4 @@
     specific language governing permissions and limitations
     under the License.
 
---%>
-<%= "openid.mode:id_res\nis_valid:false\n" %>
\ No newline at end of file
+--%><%= "openid.mode:id_res\nis_valid:false\n" %>
```

Comment:
I don't think it really works, or by luck. Because the openid4java library does not support line breaks very well. As we choose to use `\n` to make it work, we should not leave a line break depending on the operating system just before, at the beginning of the file.
But the issue I see comes from the `casOpenIdAssociationSuccessView.jsp`.